.rubocop.yml

```diff
@@ -1,14 +1,42 @@
 AllCops:
+  TargetRubyVersion: 3.0
   SuggestExtensions: false
-  NewCops: enable
-  TargetRubyVersion: 3.0.2
-  Exclude:
-    - 'vendor/**/*.rb'
-    - 'db/**/*.rb'
-    - 'bin/**/*'
+  NewCops: disable
```

Comment:
I disabled it because when I run rubocop I get a message with rubocop rules that were added but are not configured.
Something like this:
<img width="1004" alt="Screenshot 2024-03-08 at 09 25 33" src="https://github.com/Agilefreaks/ruby-interview-starting-point/assets/20145349/81faefb9-7274-4256-b1a8-04f3cfbba401">

Reply:
Yeah, you can enable them by default

Reply:
Fixed it here [d7ff898](https://github.com/Agilefreaks/ruby-interview-starting-point/pull/2/commits/d7ff89865b67ae803855b3f31a1838a86e3c12d4)